pkg/kubectl/resource/builder.go

```diff
@@ -806,12 +806,13 @@ func HasNames(args []string) (bool, error) {
 
 // MultipleTypesRequested returns true if the provided args contain multiple resource kinds
 func MultipleTypesRequested(args []string) bool {
+	if len(args) == 1 && args[0] == "all" {
+		return true
```

Comment:
> This check is somewhat misleading. In resource/builder.go, the call to ReplaceAliases replaces all with all supported resources [1], and then ResourceTypeOrNameArgs will split it into a slice of string [2]
> This means that all won't be passed in as args[0].

Although `all` will never be `args[0]` if the args slice that is passed to `MultipleTypesRequested` is built from the output of `replaceAliases` or `SplitResourceArgument`, this check is mostly for cases such as https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/get.go#L424, where arguments from a sub command are passed to it, without first going through other functions in resource/builder.go